### ISSUE TITLE
Understand current debugging options in Intel SGX

### ISSUE THREAD

**Issue by lkatalin** [enarx/enarx#481]
How do we make the enclave compatible with using `gdb`? What are the current capabilities? Can we step through instructions? We should understand this before we can address #472.

**Comment by npmccallum** [comment]
We've opened [a request](https://github.com/confidential-computing/governance/issues/51) with the CCC to merge the Open Enclave and SGX SDK ptrace code into glibc. This has spawned [an issue](https://github.com/openenclave/openenclave/issues/3064) in Open Enclave.

**Comment by platten** [comment]
No longer relevant as we've merged `gdb` support.
